docs/docsite/rst/community/reporting_bugs_and_features.rst

```diff
@@ -1,13 +1,32 @@
 Reporting Bugs And Requesting Features
 ======================================
 
-What is a bug
--------------
-Has it already been filed
-Is there a PR in flight
+I'd Like To Report A Bug
+------------------------
 
+Ansible practices responsible disclosure - if this is a security related bug, email `security@ansible.com <mailto:security@ansible.com>`_ instead of filing a ticket or posting to the Google Group and you will receive a prompt response.
 
-What's a feature
------------------
+Ansible bugs should be reported to `github.com/ansible/ansible <https://github.com/ansible/ansible>`_ after
+signing up for a free GitHub account.  Before reporting a bug, please use the bug/issue search
+to see if the issue has already been reported. This is listed on the bottom of the docs page for any module.
 
-Feature vs proposal LINK docs/community/proposal_process.rst
\ No newline at end of file
+Knowing your ansible version and the exact commands you are running, and what you expect, saves time and helps us help everyone with their issues more quickly.
```

Comment:
I know we aren't doing full content review here, though I'd just replace this with "Please complete the issue template with all requested information"